node/testing.go

```diff
@@ -27,3 +34,43 @@ func TestNode(t *testing.T, tp Type, opts ...Option) *Node {
 	require.NoError(t, err)
 	return nd
 }
+
+// WithFaultMaker is used to enable test cases with invalid blocks
```

Comment:
we also need to explicitly provide info on how the invalid blocks are created. Either it is from block N onwards. Or if it is the 1 of the M blocks produced.

Reply:
FaultMaker will be added in a different PR.